alignak/commandcall.py

```diff
@@ -113,15 +130,16 @@ def __init__(self, params, parsing=True):
             super(CommandCall, self).__init__(params, parsing=parsing)
             self.command = Command(params['command'], parsing=parsing)
 
-    def serialize(self):
+    def serialize(self, filtered_fields=None):
+        # Todo: check if necessary! Why not using the generic serialize?
         cls = self.__class__
```

Comment:
We can replace the 6 next lines with `res = super(CommandCall, self).serialize()` and kep the `res['command']...`